graphs/possible_bipartition.py

```diff
@@ -1,12 +1,56 @@
 # Can be used for BFS
 from collections import deque 
+class Graph():
 
-def possible_bipartition(dislikes):
+    def __init__(self, V):
+            self.V = V
+            self.graph = [[0 for column in range(V)] \
+                                    for row in range(V)]
+
+def possible_bipartition(self, dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(N+E)
+        Space Complexity: O(N)
     """
-    pass
-
+        colorArr = [-1] * self.V
+ 
+        # Assign first color to source
+        colorArr[dislikes] = 1
+ 
+        # Create a queue (FIFO) of vertex numbers and
+        # enqueue source vertex for BFS traversal
+        queue = []
+        queue.append(dislikes)
+ 
+        # Run while there are vertices in queue
+        # (Similar to BFS)
+        while queue:
```

Comment:
Your approach looks largely correct for an adjacency matrix, but I may be missing something without the tests